### DISCUSSION TITLE
demuxer-max-bytes

### DISCUSSION THREAD

**Discussion by Lonely233233** [hooke007/mpv_PlayKit#315] [mpv 通用]
demuxer-max-bytes=500MIB

demuxer-max-back-bytes=500MIB
这两个有什么作用

**Comment by hooke007** [comment]
你真的看不懂吗。。
https://mpv.io/manual/master/#options-demuxer-max-bytes
一个控制前向缓存量，一个控制后向

**Reply by Lonely233233** [reply]
谢谢 就是有点不明白 我知道是缓存

**Reply by hooke007** [reply]
你是不明白“缓存”还是哪个概念

**Reply by Lonely233233** [reply]
主要是demuxer-max-back-bytes向前缓存这个

**Reply by hooke007** [reply]
你的进度条在播放的时候是不断前进的，这两个值就控制相对的缓冲区间，正常人都希望有足够大的缓存量，这样在跳转的时候就能和本地文件一样瞬间完成而不需要等待下载完毕。
如果从中间开始播放，就能清晰的观察到（进度条）它在往前后两个方向缓存

**Reply by Lonely233233** [reply]
谢谢大佬 明白了
在请教一个问题demuxer-readahead-secs这个设置秒数会影响这个demuxer-max-bytes

**Reply by hooke007** [reply]
https://github.com/mpv-player/mpv/issues/6726#issuecomment-493967343

**Reply by Lonely233233** [reply]
谢谢大佬
